.konflux/tekton/bundle-e2e-pipeline.yaml

```
@@ -161,10 +170,34 @@ spec:
         volumes:
           - name: credentials
             emptyDir: { }
-        results:
-          - name: TEST_OUTPUT
-            description: Test output
+          - name: source
+            emptyDir: { }
+        stepTemplate:
+          volumeMounts:
+            - name: source
+              mountPath: /source
+            - name: credentials
+              mountPath: /credentials
         steps:
+          - name: create-repo
+            image: $(params.image)
+            script: |
+              mkdir -p /source/repo
+              chmod 777 /source/repo
+          - name: fetch-repo
+            ref:
+              resolver: http
+              params:
+                - name: url
+                  value: https://raw.githubusercontent.com/tektoncd/catalog/main/stepaction/git-clone/0.1/git-clone.yaml
```

we can use stepaction from tektoncd-catalog

```suggestion
                  value: https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/refs/heads/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
```

---

is this released ?
I see the branch name is p

---

@pramodbindal yes. This links to the `tektoncd-catalog` that fetches releases from the other repositories (`task-git` in this case). It uses the "directory based" catalog (similar to `tektoncd/catalog`) in the `p` branch, hence seeing the `p` branch in there.

It is definitely better relying on this one (the released one from `tektoncd-catalog`) than on the main branch of `task-git`. The `tektoncd-catalog` is also the source of things that are shipped as part of the openshift-pipelines operator addons.

---

Done